app/views/ministers/index.html.haml

```diff
@@ -1,13 +1,8 @@
 - breadcrumb :ministers
 
-
 %article.content-listing
   %h1
-    Ministers
-  %p.abstract
-    The Australian Government's ministers include Cabinet ministers, outer
-    ministers, assistant ministers, ministers assisting and parliamentary
-    secretaries.
```

Comment:
I think it will do once we have more ministers and sub-sections on the page. 
I'm following @coderdan's instructions as per: https://govausites.atlassian.net/browse/SITES-478